tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/fsm/FSMTrainingVectorSource.scala

```diff
@@ -34,7 +36,7 @@ abstract class FSMTrainingVectorSource(
     if (!vectorIter.hasNext) {
       resultSoFar
     } else {
-      taskHelper(resultSoFar + vectorIter.next.task, vectorIter)
+      taskHelper(resultSoFar + vectorIter.next().task, vectorIter)
     }
```

Comment:
This function confuses me. Does it do something that 'getVectorIterator.map(_.task).toSet` does not do?